reference/enterprise-requirements.rst

```diff
@@ -58,5 +59,37 @@ applies:
 |                       |                       | services              |
 +-----------------------+-----------------------+-----------------------+
 
-[note] **Note:** Three nodes are required for multi-node operation.
-[/note]
+.. note ::
+    Three nodes are required for multi-node operation.
+
+Role Based Minimum Memory Sizing
+--------------------------------
+
+For multi-node deployments the following minimum component memory sizing may be
+used to size each node based on the roles it hosts within the deployment:
+
++-----------------+----------+---------+--------------+----------+--------+-----------------+
+| Component       | Control  | Compute | Storage      | Sunbeam  | Client | Juju-Controller |
```

Comment:
Rename "Sunbeam" to "Sunbeam Controller", "Client" to "Sunbeam Client" and "Juju-Controller" to "Juju Controller" for consistency with the How-To guide, please.